spec/Vehicle/Vehicle.vspec

```diff
@@ -175,12 +175,10 @@
   description: Accumulated idle time in seconds.
 
 - Speed:
-  datatype: int32
+  datatype: float
   type: sensor
-  min: -250
-  max: 250
   unit: km/h
```

Comment:
If we agree that float is a decent type for all datapoints related to speed, then we can discuss if we should keep "km/h" or if it would make sense to shift to "m/s" (SI-units) . OEMs and others anyway needs conversion capabilities, to e.g. being able to present speed as both km/h and miles/hour.